css-layout-classes/index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="style.css" rel="stylesheet" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full"></div>
+        <h1>Choose your character!</h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-third">
+        <input
+          type="image"
+          id="link"
+          alt="NintendoCharacterLink"
+          src="images/link.png" />
```

Comment:
This should be an `img` element. Since we are not inside of a `form`, it should just be a normal `img`